cosmos/x/evm/ante/sign_mode_handler.go

```diff
@@ -22,55 +22,38 @@ package ante
 
 import (
 	"context"
-	"errors"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	signingtypes "github.com/cosmos/cosmos-sdk/types/tx/signing"
-	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+	signingv1beta1 "cosmossdk.io/api/cosmos/tx/signing/v1beta1"
+	txsigning "cosmossdk.io/x/tx/signing"
 
+	evmapi "pkg.berachain.dev/polaris/cosmos/api/polaris/evm/v1alpha1"
 	"pkg.berachain.dev/polaris/cosmos/x/evm/types"
 )
 
 // SignMode_SIGN_MODE_ETHEREUM defines the sign mode for Ethereum transactions.
 //
 //nolint:revive,stylecheck // underscores used for sign modes.
-const SignMode_SIGN_MODE_ETHEREUM signingtypes.SignMode = 42069
```

Comment:
maybe keep this constant here to avoid the ugly conversion in serializer.go

Reply:
i mean the conversion in there is definitely temporary